Sources/Sentry/include/SentrySpanOperations.h

```diff
@@ -1,5 +1,7 @@
 #import <Foundation/Foundation.h>
 
 static NSString *const SentrySpanOperationUILoad = @"ui.load";
+static NSString *const SentrySpanOperationUILoadTTID = @"ui.load.initial_display";
+static NSString *const SentrySpanOperationUILoadTTFD = @"ui.load.full_display";
```

Comment:
`l`: Again I would stick to the same naming.
```suggestion
static NSString *const SentrySpanOperationUILoadInitialDisplay = @"ui.load.initial_display";
static NSString *const SentrySpanOperationUILoadFullDisplay = @"ui.load.full_display";
```